consultation_analyser/lit/csr/questionsArchive/IaiQuestionOverview/iai-question-overview.lit.csr.mjs

```diff
@@ -102,27 +98,11 @@ export default class IaiQuestionOverview extends IaiLitBase {
                     title="Free Text Responses"
                     .total=${textResponseTotal}
                 ></iai-question-overview-subtitle>
-                
-                ${!this.responses.agreement && !this.responses.disagreement && !this.responses.unclear
-                    ? html`<p class="govuk-body">This question does not have free text responses</p>`
-                    : html`
-                        <ul class="text-response-list">
-                            <iai-text-response-item
-                                iconName="thumb_up"
-                                .text=${html`<strong>${this.responses.agreement}</strong> responses <strong>agree</strong> with the question`}
-                            ></iai-text-response-item>
-                            <iai-text-response-item
-                                iconName="thumbs_up_down"
-                                .text=${html`<strong>${this.responses.unclear}</strong> responses are <strong>unclear</strong> on whether agree or disagree with the question`}
-                            ></iai-text-response-item>
-                            <iai-text-response-item
-                                iconName="thumb_down"
-                                .text=${html`<strong>${this.responses.disagreement}</strong> responses <strong>disagree</strong> with the question`}
-                            ></iai-text-response-item>
-                        </ul>
-                    `
-                }
 
+              ${!this.responses.count
+                ? html`<p class="govuk-body">This question does not have free text responses</p>`
+                : ''
+              }
```

Comment:
@ekin-odabas I feel like there should be a non-ternary way to do this - is there a better JS approach to this?